pipelines/broad/dna_seq/germline/joint_genotyping/exome/test_inputs/Plumbing/gather_vcfs_high_memory.json

```diff
@@ -28,13 +28,15 @@
   "JointGenotyping.haplotype_database": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.haplotype_database.txt",
 
   "JointGenotyping.callset_name": "small_callset_low_threshold",
-  "JointGenotyping.sample_name_map" : "gs://broad-gotc-test-storage/joint_genotyping/exome/plumbing/callset/plumbing_sample_map",
+  "JointGenotyping.sample_name_map" : "gs://broad-gotc-test-storage/joint_genotyping/exome/plumbing/callset/reblocked_plumbing_sample_map",
 
   "JointGenotyping.small_disk" : 100,
   "JointGenotyping.medium_disk" : 200,
   "JointGenotyping.large_disk" : 500,
   "JointGenotyping.huge_disk" : 2000,
 
   "JointGenotyping.gather_vcfs": true,
-  "JointGenotyping.snps_variant_recalibration_threshold": 50
+  "JointGenotyping.snps_variant_recalibration_threshold": 50,
```

Comment:
In, gather_vcfs_low_memory.json you have
"JointGenotyping.snps_variant_recalibration_threshold": 500000

Is that on purpose?  If so, feel free to ignore this comment... I'm just pointing out something that looked out of place to me.

Reply:
This is expected.  We're running the joint genotyping workflow four different ways: {gather, don't gather} x {classic, VQSR with parallel scoring and tranches}.  The 50 make this look like a "big" callset because we have ~53 samples, so it changes the control flow to use the path with parallel VQSR scoring.